src/backend/parser/parse_relation.c

```diff
@@ -604,7 +619,6 @@ scanRTEForColumn(ParseState *pstate, RangeTblEntry *rte, char *colname,
 			return result;
 
 		/* quick check to see if name could be a system column */
```

Comment:
this comment aslo should be moved up

Reply:
Not sure this comment is still relevant, the check is not quick anymore, etc, but ok, moved.